metaserver/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -325,21 +318,6 @@ object ScalametaLanguageServer extends LazyLogging {
     subscriber -> semanticdbPublisher
   }
 
-  def interactiveSemanticdbStream(cwd: AbsolutePath)(
```

Comment:
Did you remove this entirely because you think there's a better approach? Otherwise just commenting/removing the code that invokes this should be enough.

Reply:
I think There may be a better way to model this as a stream from the notifications, want to experiment with it today